### PR TITLE
feat: auto expose by interpolate tokens

### PR DESCRIPTION
**Pull request by Gvozd**
for example

```
module: {
  loaders: [
    { test: /\.js$/, loader: "expose?[name]" }
  ]
}
```

### CODE REVIEW

**Comment by Gvozd** [comment]
@SpaceK33z , please look and merge this PR
Or maybe has any remarks?
I may fix it, if need

File: package.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "name": "expose-loader",
-  "version": "0.7.1",
+  "version": "0.8.0",
```
Comment (SpaceK33z) [inline package.json]: Please don't bump the version yourself, this is the job of the maintainer.

**Comment by Gvozd** [comment]
@SpaceK33z , I rebase(for clear history) & edit commit, as you request

**Comment by raoulus** [comment]
:+1: 

Would be great to get this merged

**Comment by michael-ciniawsky** [comment]
@Gvozd Sry for the inconvenience but could close and reooen the PR to trigger the CLA bot again and rebase aginast current master ? ⚠️ Please also take a look at #39 and maybe wait until it gets merged 😛

**Comment by d3viant0ne** [comment]
Need a signed CLA and a `rebase` before we can do anything with it.

**Comment by Gvozd** [comment]
@michael-ciniawsky , OK i wait for #39, and rebase

**Comment by raoulus** [comment]
any news on this PR? :)

**Comment by Gvozd** [comment]
@raoulus , I wait for #39

**Comment by d3viant0ne** [comment]
If we are waiting for #39 which in turn is going in with defaults, this needs to be in the 1.0.0 milestone. @michael-ciniawsky ?

**Comment by Gvozd** [comment]
I did not wait

**Pull request closed without merging**